cf_xarray/accessor.py

```diff
@@ -653,6 +647,9 @@ class CFAccessor:
 
     def __init__(self, da):
         self._obj = da
+        self._defined_cell_measures = tuple(
```

Comment:
This will take time for datasets with lots of variables every time the accessor is created.

Let's instead do

``` python
self._defined_cell_measures=None
```
and use the following when we do actually need the measure names

```python
def _get_all_cell_measures(self):
    if not self._defined_cell_measures:
        self._defined_cell_measures = set(_CELL_MEASURES + tuple(self.cell_measures))

    return self._defined_cell_measures
```

Reply:
Good point. See 36de5e3400e4106696c31480b407546f58dc8c6e